k8s/myeventsappchart/values.yaml

```diff
@@ -32,23 +32,23 @@ databaseClusterIpService:
     targetPort: 5432
   type: ClusterIP
 dbConfigmap:
-  dbHost: database-cluster-ip-service
+  dbHost: myapp-database-cluster-ip-service
```

Comment:
@qimolin I had to hardcode "myapp", couldn't figure out how to do it another way. But this fixes the error:

>failed to connect to `host=database-cluster-ip-service ... no such host)